### DISCUSSION TITLE
data understanding

### DISCUSSION THREAD

**Discussion by w-i-z-a-r-d** [theOehrly/Fast-F1#171] [Q&A]
what is the difference between Driver.telemetry['Time'] and driver_car_data['Time'] after we do get_car_data() for a specific lap for both ?

**Comment by w-i-z-a-r-d** [comment]
it seems that the Telemetry function contains more data than the get_car_data, comparing apple to apple, meaning speed vs speed, RPM vs RPM. right?

**Reply by theOehrly** [reply]
See https://theoehrly.github.io/Fast-F1/core.html#fastf1.core.Laps.get_telemetry

Because car and position data do not use a common timebase, there are more values than before after merging them both and interpolating (timestamps from both are kept). More data does mean potentially LESS accuracy here, though, as the values are interpolated.

If you filter the merged data from `get_telemetry` using the source column, you can get the original values.
